docs/unpacked-format.rst

```diff
@@ -0,0 +1,120 @@
+..  _unpacked-format:
+
+Structure of Unpacked Experiments
+*********************************
+
+While *reprounzip* is designed to allow users to reproduce an experiment without having to master the tool used to run it (e.g.: `Vagrant <https://www.vagrantup.com/>`__ and `Docker <https://www.docker.com/>`__), in some situations it might be useful to go behind the scenes and interact with the unpacked experiments directly.
+
+This page describes in more details how the unpackers operate.
+
+..  note:: Future versions of unpackers might work in a different way. No attempt is made to make unpacked experiments compatible across different versions of *reprounzip*.
+
+..  _unpacked-directory:
+
+The `directory` Unpacker
+========================
+
+The experiment directory contains:
+
+* The original configuration file ``config.yml``.
+* The pickle file ``.reprounzip``.
```

Comment:
No, nothing specific is stored there, although I should probably add a paragraph about how input files are handled for all the unpackers.